subworkflows/nf-core/bam_variant_demix_boot_freyja/main.nf

```diff
@@ -0,0 +1,75 @@
+include { FREYJA_VARIANTS   } from '../../../modules/nf-core/freyja/variants'
+include { FREYJA_UPDATE     } from '../../../modules/nf-core/freyja/update'
+include { FREYJA_DEMIX      } from '../../../modules/nf-core/freyja/demix'
+include { FREYJA_BOOT       } from '../../../modules/nf-core/freyja/boot'
+
+workflow BAM_VARIANT_DEMIX_BOOT_FREYJA {
+
+    take:
+    ch_bam          // channel: [ val(meta), [ /path/to/bam ] ]
+    fasta           // channel: [ val(meta), [path/to/reference.fasta] ]
+    repeats         // value repeats
+    db_name         // string db_name
+    barcodes        // /path/to/barcodes.csv
+    lineage_meta    // /path/to/lineage_metadata
```

Comment:
```suggestion
    ch_bam          // channel: [ val(meta), path(bam) ]
    ch_fasta        // channel: [ val(meta), path(fasta) ]
    val_repeats     // value repeats
    val_db_name     // string db_name
    barcodes        // channel: [ val(meta), path(barcodes) ]
    lineage_meta    // channel: [ val(meta), path(lineage_meta) ]
```

Please also add the meta value to the modules using barcodes and lineage meta :)